WalletWasabi.Gui/Controls/WalletExplorer/SendTabView.xaml

```diff
@@ -67,7 +67,7 @@
                                       UseFloatingWatermark="True"
                                       MinWidth="124"
                                       MinHeight="46" />
-            <StackPanel>
+            <StackPanel IsVisible="{Binding !IsCustomFee}">
```

Comment:
Yeah that could be pretty easy to implement, would make the logic, attempt to use the custom fee if there is text in the box, otherwise use the slider